src/main/command/statuscheckcommands/StatusCheckCommand.ts

```diff
@@ -10,14 +11,18 @@ export class StatusCheckCommand extends Command {
                    userPerms: Permissions,
                    messageReply: Function,
                    ...args: CommandArgs[]): CommandResult {
-
+        //Calculation for converting Discord Uptime (milliseconds) into hours, minutes and seconds.
         const { uptime } = args[0] as StatusCheckCommandData;
+        //Hours
         let upTimeInHours = (uptime/1000) / 3600;
         let rupTimeInHours = Math.floor(upTimeInHours);
+        //Minutes
         let upTimeInMinutes = (upTimeInHours - rupTimeInHours) * 60;
         let rupTimeInMinutes = Math.floor(upTimeInMinutes);
+        //Seconds
         let upTimeInSeconds = (upTimeInMinutes - rupTimeInMinutes) * 60;
         let rupTimeInSeconds = Math.floor(upTimeInSeconds);
+        //Message Output
         messageReply(`Keesiao Bot has gone keesiao for ${rupTimeInHours} hours, ${rupTimeInMinutes} minutes and ${rupTimeInSeconds} seconds.`);
```

Comment:
Can the reply be "Uptime: x hours, y minutes and z seconds". I feel that keesiao bot has gone keesiao part is unnecessary.